packages/Liquid-Tests.package/ObjectRepoTests.class/instance/testifPresentIfAbsentRemote.st

```diff
@@ -0,0 +1,22 @@
+tests
+testifPresentIfAbsentRemote
+	| server port url client presentCheck|
+	presentCheck := false.
+	port := 10000 random.
+	server := RemoteRepoServer new listenOn: port.
+	url := 'http://localhost:' , port.
+	client := RemoteRepoClient new url: url.
```

Comment:
[sand] This test is very similar to the one with the LocalObjectRepo - maybe we can extract the common body into a method and inject the `LocalObjectRepo` / `RemoteRepoClient` instance via parameters?

Reply:
Good thought. I have set up setUp and tearDown methods that manage this and the `server destroy`

Reply:
Nice! We still have some duplicated code in the remote / local test, can we extract them in some method?

Reply:
I have combined the two tests into one in the next commit